### PR TITLE
OpenZFS 8072 - zfs-tests: several test cases incorrectly spell TESTPOOL

### PR DESCRIPTION
**Pull request by dinatale2**
Authored by: Yuri Pankov <yuri.pankov@nexenta.com>
Reviewed by: Matthew Ahrens <mahrens@delphix.com>
Reviewed by: John Kennedy <jwk404@gmail.com>
Approved by: Richard Lowe <richlowe@richlowe.net>
Ported-by: Giuseppe Di Natale <dinatale2@llnl.gov>

OpenZFS-issue: https://www.illumos.org/issues/8072
OpenZFS-commit: https://github.com/openzfs/openzfs/commit/56e4733

### CODE REVIEW

File: tests/zfs-tests/tests/functional/cli_user/misc/misc.cfg [hunk 1]
```diff
@@ -67,17 +67,17 @@ else
...
+		passthrough	discard		on		 \
 		fletcher2	on				on \
 		on		legacy		none		on \
 		128k		none		on		on \
 		visible"
 
 	# these are an alternate set of property values
 	PROP_ALTVALS="\
-		noallow		noacl		off		 \
+		passthrough-x	noacl		off		 \
```
Comment (dinatale2) [inline tests/zfs-tests/tests/functional/cli_user/misc/misc.cfg]: This line in the openzfs version reads `passthrough-x groupmask....`. I was going to make this line completely consistent with upstream.

Comment (dinatale2) [inline tests/zfs-tests/tests/functional/cli_user/misc/misc.cfg]: Turns out that line last changed in 2013. Went ahead and pulled in the `groupmask` change to make it consistent.

**Comment by behlendorf** [comment]
@dinatale2 can you investigate why the updated test cases are failing.

**Comment by dinatale2** [comment]
Sorry for leaving this alone for a while, looking at it again today.

**Pull request closed without merging**